src/cycax/cycad/assembly.py

```diff
@@ -79,9 +88,9 @@ def merge(self, part1: CycadPart, part2: CycadPart):
             ValueError: if the sizes of the parts are not identical.
         """
         if (
```

Comment:
Nothing wrong here just want to make sure you know about all an alternative to this is:
```
      if (all(
            part1.x_size == part2.x_size,
            part1.y_size == part2.y_size,
            part1.z_size == part2.z_size)
        ):
```
A tip for when you have longer than 3 compares. The and method has better performance than the all way.